fapi-client/src/test/scala/com/gu/facia/api/utils/FaciaContentUtilsTest.scala

```diff
@@ -0,0 +1,357 @@
+      FaciaContentUtils.mediaType(latestSnap) should be(Some(Gallery))
+    }
+
+    "should return a Audio for a Content with type/audio in LatestSnap" in {
+      val latestSnap = makeLatestSnap("some-latest-snap", contentWithTagIds("type/audio"))
+      FaciaContentUtils.mediaType(latestSnap) should be(Some(Audio))
+    }
+
+    "should return a Video for a Content with type/video in LatestSnap" in {
+      val latestSnap = makeLatestSnap("some-latest-snap", contentWithTagIds("type/video"))
+      FaciaContentUtils.mediaType(latestSnap) should be(Some(Video))
+    }
+
+    "should return a Gallery for a Content with type/gallery in CuratedContent" in {
+      val cuaratedContent = makeLatestSnap("some-curated-content", contentWithTagIds("type/gallery"))
+      FaciaContentUtils.mediaType(cuaratedContent) should be(Some(Gallery))
+    }
+
+    "should return a Audio for a Content with type/audio in CuratedContent" in {
+      val cuaratedContent = makeLatestSnap("some-curated-content", contentWithTagIds("type/audio"))
+      FaciaContentUtils.mediaType(cuaratedContent) should be(Some(Audio))
+    }
+
+    "should return a Video for a Content with type/video in CuratedContent" in {
+      val cuaratedContent = makeLatestSnap("some-curated-content", contentWithTagIds("type/video"))
+      FaciaContentUtils.mediaType(cuaratedContent) should be(Some(Video))
+    }
+
+    "should return a Gallery for a Content with type/gallery in SupportingCuratedContent" in {
+      val supportingCuratedContent = makeLatestSnap("some-supporting-curated-content", contentWithTagIds("type/gallery"))
+      FaciaContentUtils.mediaType(supportingCuratedContent) should be(Some(Gallery))
+    }
+
+    "should return a Audio for a Content with type/audio in SupportingCuratedContent" in {
+      val supportingCuratedContent = makeLatestSnap("some-supporting-curated-content", contentWithTagIds("type/audio"))
+      FaciaContentUtils.mediaType(supportingCuratedContent) should be(Some(Audio))
+    }
+
+    "should return a Video for a Content with type/video in SupportingCuratedContent" in {
+      val supportingCuratedContent = makeLatestSnap("some-supporting-curated-content", contentWithTagIds("type/video"))
+      FaciaContentUtils.mediaType(supportingCuratedContent) should be(Some(Video))
+    }
+  }
+
+  "isLive" - {
+    def contentWithFields(fields: Map[String, String]) = Content(
+      "content-id", Some("section"), Some("Section Name"), Option(staticDateTime), "webTitle", "webUrl", "apiUrl",
+      fields = Option(fields),
+      Nil, None, Nil, None)
+
+    "should return a false for a LinkSnap" in {
+      val linkSnap = makeLinkSnap("some-link-snap")
+      FaciaContentUtils.isLive(linkSnap) should be (false)
+    }
+
+    "should return true and false on inner content for LatestSnap" in {
+      val latestSnap = makeLatestSnap("some-latest-snap", contentWithFields(Map("liveBloggingNow" -> "true")))
```

Comment:
Maybe. What is testing really? Strictly one assert per test case or grouping another way?